tests/integrations/config/application.py

```diff
@@ -1,14 +1,15 @@
-import os
+from masonite.environment import env
 
-KEY = os.getenv("APP_KEY", "-RkDOqXojJIlsF_I8wWiUq_KRZ0PtGWTOZ676u5HtLg=")
+KEY = env("APP_KEY", "-RkDOqXojJIlsF_I8wWiUq_KRZ0PtGWTOZ676u5HtLg=")
 
+DEBUG = env("APP_DEBUG", True)
```

Comment:
here for existing projects, it won't raise error it will be just be false.

Reply:
if it will always be false for existing projects, how do they enable it?

Reply:
you're right ! Here is what we can do until a major release (maybe): https://github.com/MasoniteFramework/masonite/pull/506/files/f91ba5df468a9fb11a34af5a3913190c4d97788b#r790302614